lib/src/from_handlers.dart

```diff
@@ -67,12 +67,9 @@ class _StreamTransformer<S, T> implements StreamTransformer<S, T> {
       controller.onResume = () => subscription?.resume();
```

Comment:
You shouldn't need the `?.` here. The `subscription` is set to non-null in `onListen` above, and is only set to `null` in the `onCancel` callback. There will not be a call to `onPause` or `onResume` before a call to `onListen` or after a call to `onCancel`.
(You can even wait and only set the `onPause`/`onResume`/`onCancel` callbacks inside the `onListen`, they won't be called before that one returns, and then you can do:
   controller.onPause = subscription.pause;
   controller.onResume = subscription.resume;
without the extra function wrapping.)